app/models/publisher.rb

```diff
@@ -112,8 +112,7 @@ class Publisher < ApplicationRecord
   ###############################
 
   scope :uphold_selected_provider, -> {
-    joins("INNER JOIN uphold_connections
-           ON uphold_connections.id = publishers.selected_wallet_provider_id
+    where("uphold_connections.id = publishers.selected_wallet_provider_id
```

Comment:
Unless I'm mistaken, scopes like these should still work in isolation.

Reply:
Yeah these are only ever called from our job, and with eager loading it throws a double reference error to have the join here. I'll work on a workaround.

Reply:
Fixed.

Reply:
So the confusing part is that below, you're using both `includes` and `joins`. Is it possible to revert back to the original definition of `uphold_selected_provider`, and if not, why?

Reply:
You need to use `includes` for eager loading since we enforce strict loading, but we need to `joins` for filtering.